rosidl_generator_c/CMakeLists.txt

```diff
@@ -8,10 +8,11 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
+find_package(ament_cmake_ros REQUIRED)
```

Comment:
With `ament_cmake_ros` being added you can remove `ament_cmake` (which is a transitive dependency) to keep the CMake file simpler.